TERMINAL LIST 1
Type:      File
Name:      f1
Original:  <dir>/r5/f1
Created:   2020-05-31 12:00:00 

Type:      File
Name:      f2
Original:  <dir>/r5/f2
Created:   2020-05-31 12:00:00 

Type:      File
Name:      f5
Original:  <dir>/r5/f5
Created:   2020-05-31 12:00:00 
Entries:      3
Total Files:  3
Log size:     0.00B
Size:         20.00B 


EXCEPTION RESTORE
Encountered exception(s)

- Path not found: f4

- Path not found: f3



CallStack (from HasCallStack):
  throwCS, called at src/SafeRm.hs:269:5 in safe-rm-0.1-inplace:SafeRm

LOGS RESTORE
[2020-05-31 12:00:00][functional.restore][Debug] Trash home: <dir>/r5/.trash
[2020-05-31 12:00:00][functional.restore.readIndex][Debug] Index path: <dir>/r5/.trash/.index.csv
[2020-05-31 12:00:00][functional.restore.readIndex.readIndexWithFold][Debug] Found: MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "f1"}, originalPath = MkPathI {unPathI = "<dir>/r5/f1"}, created = MkTimestamp {unTimestamp = 2020-05-31 12:00:00}}
[2020-05-31 12:00:00][functional.restore.readIndex.readIndexWithFold][Debug] Found: MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "f2"}, originalPath = MkPathI {unPathI = "<dir>/r5/f2"}, created = MkTimestamp {unTimestamp = 2020-05-31 12:00:00}}
[2020-05-31 12:00:00][functional.restore.readIndex.readIndexWithFold][Debug] Found: MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "f5"}, originalPath = MkPathI {unPathI = "<dir>/r5/f5"}, created = MkTimestamp {unTimestamp = 2020-05-31 12:00:00}}
[2020-05-31 12:00:00][functional.restore.restoring][Debug] MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "f5"}, originalPath = MkPathI {unPathI = "<dir>/r5/f5"}, created = MkTimestamp {unTimestamp = 2020-05-31 12:00:00}}
[2020-05-31 12:00:00][functional.restore.restoring][Debug] MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "f2"}, originalPath = MkPathI {unPathI = "<dir>/r5/f2"}, created = MkTimestamp {unTimestamp = 2020-05-31 12:00:00}}
[2020-05-31 12:00:00][functional.restore.restoring][Debug] MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "f1"}, originalPath = MkPathI {unPathI = "<dir>/r5/f1"}, created = MkTimestamp {unTimestamp = 2020-05-31 12:00:00}}
[2020-05-31 12:00:00][functional.restore][Info] Restored: <dir>/r5/f5, <dir>/r5/f2, <dir>/r5/f1
[2020-05-31 12:00:00][functional][Error] Encountered exception(s)

- Path not found: f4

- Path not found: f3



CallStack (from HasCallStack):
  throwCS, called at src/SafeRm.hs:269:5 in safe-rm-0.1-inplace:SafeRm


TERMINAL LIST 2
Entries:      0
Total Files:  0
Log size:     0.00B
Size:         5.00B 


